app/assets/stylesheets/content/_flash_messages.sass

```diff
@@ -42,6 +42,7 @@
 
 .flash
   display: none
+  padding: 15px 5px
```

Comment:
disagree for two reasons:
1. was just "4px" before
2. why should font size "just" change

besides that I very much dislike the current style of flash messages and a bit more of padding makes them a lot nice.